src/components/Modal/Modal.js

```diff
@@ -1,19 +1,32 @@
-import React from "react";
+import React, { useState } from "react";
 import { FaTrash } from "react-icons/fa";
 import { useDispatch, useSelector } from "react-redux";
 import PlaylistInputForm from "./PlaylistInputForm";
-import { deletePlaylist } from "../../redux/reducers/playlist/playlistSlice";
+import {
+  deletePlaylist,
+  addVideoToPlaylist,
+} from "../../redux/reducers/playlist/playlistSlice";
 import { toast } from "react-toastify";
-export default function Modal({ setShowModal }) {
+export default function Modal({ setShowModal, video }) {
+  console.log(setShowModal);
   const playlistsData = useSelector((state) => state.playlists.playlists);
   const isUserLoggedIn = useSelector((state) => state.auth.isUserLoggedin);
+  const [currentPlaylist, setCurrentPlaylist] = useState({});
   console.log(playlistsData);
   const playlistDispatch = useDispatch();
   const deletePlaylistName = async (playlistId) => {
     if (!isUserLoggedIn) return toast.error("Login to delete playlist");
     await playlistDispatch(deletePlaylist(playlistId)).unwrap();
     return toast.success("Removed playlist successfully!");
   };
+
+  const addVideoToPlayList = async (video, playlistId) => {
+    if (!isUserLoggedIn)
+      return toast.error("Please login to add video to playlist");
```

Comment:
I think it will be better if you navigate the user to login page instead of showing the toast. Just a suggestion!

Reply:
cool, noted!